src/app/analytics/services/analytics.service.ts

```diff
@@ -93,40 +93,33 @@ export function identifyPlan(newValue: number) {
   }
 }
 
-export function trackSignOut() {
-  /* window.analytics.track(AnalyticsTrack.SignOut);
-  window.analytics.reset(); */
+export function rudderanalyticsSignOut() {
+  window.rudderanalytics.track('User Logout');
 }
 
-export function trackSignIn(payload: { email: string; userId: string }): void {
-  // window.analytics.track(AnalyticsTrack.SignIn, payload);
+export function rudderanalyticsSignIn(uuid: string, email: string): void {
```

Comment:
I would advise receiving the payload as an object so we are not bound to the order of the parameters, we already experimented with tracks received with these two parameters interchanged.